enterprise/views.py

```diff
@@ -774,6 +782,9 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             organization_logo = None
             organization_name = None
 
+        if enterprise_course_enrollment and not enterprise_course_enrollment.consent_granted:
+            add_consent_declined_message(request, enterprise_customer, course_details)
```

Comment:
I agree with Zubair's point - I know we are initially using this with course level consent but it seems like it wouldn't be too difficult to check the UserDataSharingConsentAudit record if it exists.